less/_defaults/_spacing-containers.less

```diff
@@ -29,27 +29,27 @@
 @page-padding-hoz: 0;
 
 @page-header-padding-ver: 2rem;
-@page-header-padding-hoz: .5rem;
+@page-header-padding-hoz: 1rem;
 
 // Article header padding
 // --------------------------------------------------
 @article-header-padding-ver: 0;
-@article-header-padding-hoz: .5rem;
+@article-header-padding-hoz: 1rem;
 
 // Block padding
 // --------------------------------------------------
 @block-padding-ver: 4rem;
-@block-padding-hoz: .5rem;
+@block-padding-hoz: 1rem;
 
 // Component padding
 // --------------------------------------------------
-@component-padding-ver: .5rem;
-@component-padding-hoz: .5rem;
+@component-padding-ver: 0.5rem;
```

Comment:
Can this value be increased to 1rem to ensure component padding is equal in all directions

Reply:
I did look at this, but increasing to 1rem make it look untidy to my eyes. Will update though